src/java.base/unix/native/libjava/ProcessImpl_md.c

```diff
@@ -317,10 +317,21 @@ releaseBytes(JNIEnv *env, jbyteArray arr, const char* parr)
         (*env)->ReleaseByteArrayElements(env, arr, (jbyte*) parr, JNI_ABORT);
 }
 
-#define IOE_FORMAT "error=%d, %s"
+#define IOE_FORMAT "error=%d, %s%s"
+
+#define SPAWN_HELPER_INTERNAL_ERROR_MSG "\n" \
+  "Possible reasons:\n" \
+  "  - Spawn helper ran into JDK version or configuration problems\n" \
```

Comment:
Can we be clear and say "Spawn helper ran into a mismatch between the running JDK and the version on disk" or something similar?

Reply:
I just want to point uses to know what the root cause might be and not have to reach out through support channels.
Does "Spawn helper ran into JDK version mismatch or configuration problems" seem better to you?

Reply:
I switched to "Spawn helper ran into JDK version mismatch", which I think is straight up the most plausible cause.